advanced/attributes/src/main/java/org/arakhne/afc/attrs/attr/AttributeValueImpl.java

```diff
@@ -760,13 +760,8 @@ public void castAndSet(AttributeType attrType, Object attrValue) {
 		try {
 			if (attrValue instanceof AttributeValue) {
 				this.type = ((AttributeValue) attrValue).getType();
-				try {
-					this.value = ((AttributeValue) attrValue).getValue();
-					this.assigned = true;
-				} catch (AttributeNotInitializedException e) {
-					this.value = attrType.getDefaultValue();
-					this.assigned = true;
-				}
+				this.value = ((AttributeValue) attrValue).getValue();
```

Comment:
This contribution breaks the expected behavior of the cast&set function. It will be ignored in the merging action.